packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -515,16 +425,12 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 		<LoggedInPlansFeatureActionButton
 			planSlug={ planSlug }
 			availableForPurchase={ availableForPurchase }
-			onCtaClick={ onCtaClick }
+			onCtaClick={ callback }
 			currentSitePlanSlug={ currentSitePlanSlug }
 			buttonText={ buttonText }
```

Comment:
`buttonText` is another candidate for moving out. As far as I checked, they are basically `text` overrides that can be passed through `useAction` hook instead

Let's keep in mind if that's the case

Reply:
Yes that was my understanding as well 👍

I'll look into lifting it out of the `actions` component

Reply:
[7983665](https://github.com/Automattic/wp-calypso/pull/89926/commits/79836654f31e0500533c2da85af57a0798177bd9)